spotbugs/src/test/java/edu/umd/cs/findbugs/util/MutableClassesTest.java

```diff
@@ -14,6 +14,10 @@ public void TestKnownImmutable() {
         Assert.assertFalse(MutableClasses.mutableSignature("Ljava/lang/String;"));
     }
 
+    public void TestNamedImmutable() {
```

Comment:
I guess we need an annotation here?

```suggestion
    @Test
    public void TestNamedImmutable() {
```